client/src/js/app.js

```diff
@@ -178,18 +179,19 @@ function bhimaconfig($routeProvider) {
     controller: 'budget',
     templateUrl: 'partials/budget/budget.html'
   })
-
+  
+  /* TODO Depricate/ remove */
   /* journal routes */
 
-  .when('/journal', {
+  .when('/1.X-journal', {
```

Comment:
We do not - I left these in case anyone wanted to reference the previous posting journal structure; this can easily be done using git though!